src/applications/widget-editor/src/components/footer/component.js

```diff
@@ -22,26 +22,9 @@ const StyledFooter = styled.footer`
   justify-content: space-between;
 `;
 
-const Footer = ({ enableSave, enableInfo, onSave }) => {
-  const [modalOpen, setModalOpen] = useState(false);
-
+const Footer = ({ enableSave, onSave }) => {
   return (
-    <StyledFooter disabled={!enableSave && !enableInfo}>
-      {enableInfo && (
-        <Fragment>
-          <Modal isOpen={modalOpen} closeModal={() => setModalOpen(false)}>
```

Comment:
@pablopareja remove this modal aswel, its only used here. If we need it later we can add it back. This includes these hooks `modalOpen` `setModalOpen`

Reply:
Done here: https://github.com/Vizzuality/widget-editor/pull/74/commits/ef93d4e785d09fdbfdba14e152b5bab209975b59 ;)